pkg/image/registry/imagestreamtag/rest.go

```diff
@@ -115,8 +115,12 @@ func (r *REST) Update(ctx kapi.Context, tagName string, objInfo rest.UpdatedObje
 			return nil, false, kapierrors.NewBadRequest("namespace is required on ImageStreamTags")
 		}
 		imageStream = &api.ImageStream{
-			ObjectMeta: kapi.ObjectMeta{Namespace: namespace, Name: name},
+			ObjectMeta: kapi.ObjectMeta{
+				Namespace: namespace,
+				Name:      name,
+			},
 		}
+		kapi.FillObjectMetaSystemFields(ctx, &imageStream.ObjectMeta)
```

Comment:
I don't know what this is doing... not sure we have createonupdate for other virtual resources